### PR TITLE
 More reliable reference to native Symbol 

### PR DESCRIPTION
**Pull request by cyborgx37**
Some bundlers may be configured to replace unqualified references to Symbol with a polyfill. Accessing Symbol as a property of the global is more reliable.

See issue #29

### CODE REVIEW

**Comment by cyborgx37** [comment]
Woot for the awesome automated test suite. I'll look into fixing this shortly.

**Comment by cyborgx37** [comment]
@medikoo I believe that this is now ready for review. Please feel free to recommend changes and I will get to them as soon as I can.

File: native-symbol.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Returns the native Symbol, if there is one.
```
Comment (medikoo) [inline native-symbol.js]: I think I wouldn't create this module, instead I would just refer to `global` in all modules.
It just serves a shortcut for `global.Symbol` which seems not perfectly justified.

Other issue with it, is that if we assign polyfill to `global` then it'll return a _polyfill_ and not _native_ symbol as name implies

**Review by medikoo — CHANGES_REQUESTED** [review]
Great thanks @cyborgx37 It looks really good. I've added just one remark

File: native-symbol.js (hunk 1 above)
Comment (cyborgx37) [reply native-symbol.js]: Ok, I can do that, but I was trying to keep the line count minimal. Using global everywhere will require a larger PR.

Comment (cyborgx37) [reply native-symbol.js]: On further reflection, I think I can do this with fewer line changes than I was originally thinking.

**Comment by cyborgx37** [comment]
This is ready for re-review.

**Review by medikoo — APPROVED** [review]
Thank you, looks great!

**Pull request merged**